drivers/SmartThings/matter-window-covering/src/init.lua

```diff
@@ -41,6 +41,15 @@ end
 
 local function match_profile(device)
   local profile_name = "window-covering"
+  local lift_eps = device:get_endpoints(clusters.WindowCovering.ID, {feature_bitmap = clusters.WindowCovering.types.Feature.LIFT})
+  local tilt_eps = device:get_endpoints(clusters.WindowCovering.ID, {feature_bitmap = clusters.WindowCovering.types.Feature.TILT})
+  if #tilt_eps > 0 then
+    if #lift_eps > 0 then
+      profile_name = profile_name .. "-tilt"
+    else
+      profile_name = profile_name .. "-tilt-only"
+    end
+  end
```

Comment:
```suggestion
  local lift_eps = device:get_endpoints(clusters.WindowCovering.ID, {feature_bitmap = clusters.WindowCovering.types.Feature.LIFT})
  local tilt_eps = device:get_endpoints(clusters.WindowCovering.ID, {feature_bitmap = clusters.WindowCovering.types.Feature.TILT})
  
  if #tilt_eps > 0 then
    profile_name = profile_name .. "-tilt"
    if #[lift_eps](device:get_endpoints(clusters.WindowCovering.ID, {feature_bitmap = clusters.WindowCovering.types.Feature.LIFT})) == 0 then
      profile_name = profile_name .. "-only"
    end
  end
```

would this be more future-proof against further configure logic changes? not sure

Reply:
Good idea! I made this change in d5424d31248d3d81549e817624f8905565ec15da. I also realized that the battery check was going to overwrite the profile name rather than append to it so I fixed that as well.